src/NavItem.js

```diff
@@ -5,14 +5,14 @@ const NavItem = ({
   divider,
   children,
   href = '',
+  onClick,
   ...props
 }) => {
   if (divider) return <li className='divider' />;
+  const a = onClick ? <a>{children}</a> : <a href={href}>{children}</a>;
   return (
-    <li {...props}>
-      <a href={href}>
-        { children }
-      </a>
+    <li {...props} onClick={onClick}>
+      {a}
```

Comment:
why don't we add the `onClick` to the `a` tag?

Reply:
That would be simplier =)

I will adjust it.